bower.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "bootstrap",
   "description": "The most popular front-end framework for developing responsive, mobile first projects on the web.",
-  "version": "3.3.4",
```

Comment:
@byronmansfield As noted in both the commit message and pull request description, see https://github.com/bower/bower.json-spec/commit/a325da3d79baab018c572d75dc1781b12322f6cd